### PR TITLE
Impl list lookup

### LINKED ISSUE #67: List lookup

### PR DESCRIPTION
**Pull request by mmmommm**
close #67

### CODE REVIEW

File: list.go [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+}
+
+type OwnedListsLookUpByIDResponse struct {
+	Lists    []*List             `json:"data"`
+	Includes *ListIncludes       `json:"includes,omitempty"`
+	Errors   []*APIResponseError `json:"errors,omitempty"`
+	Meta     *ListMeta           `json:"meta"`
+	Title    string              `json:"title,omitempty"`
+	Detail   string              `json:"detail,omitempty"`
+	Type     string              `json:"type,omitempty"`
```
Comment (sivchari) [inline list.go:42]: Please tell me reproduce way to display these.

File: list_lookup_test.go [hunk 2]
```diff
@@ -0,0 +1,358 @@
...
+	"io"
+	"net/http"
+	"strings"
+	"testing"
+
+	"github.com/google/go-cmp/cmp"
+	"github.com/sivchari/gotwtr"
+)
+
+func Test_lookUpOwnedListByID(t *testing.T) {
```
Comment (sivchari) [inline list_lookup_test.go:14]: add `t.Parallel()`

File: list.go [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+
+type ListField string
+
+const (
+	ListFieldCreatedAt   ListField = "created_at"
+	ListFieldPrivate     ListField = "private"
+	ListFollowerCount    ListField = "follower_count"
+	ListMemberCount      ListField = "member_count"
+	ListOwnerID          ListField = "owner_id"
+	ListFieldDescription ListField = "description"
```
Comment (sivchari) [inline list.go:11]:
```suggestion
	ListFieldCreatedAt   ListField = "created_at"
	ListFollowerCount    ListField = "follower_count"
	ListMemberCount      ListField = "member_count"
	ListFieldPrivate     ListField = "private"
	ListFieldDescription ListField = "description"
	ListOwnerID          ListField = "owner_id"
```

**Review by sivchari — CHANGES_REQUESTED** [review]
Please add `owner_id` to expansiong.go

File: list_lookup_test.go (hunk 2 above)
Comment (mmmommm) [reply list_lookup_test.go:14]: I think `t.Parallel()` is already writing

File: list_lookup_test.go [hunk 4]
```diff
@@ -0,0 +1,358 @@
...
+				Detail: "One or more parameters to your request was invalid.",
+				Type:   "https://api.twitter.com/2/problems/invalid-request",
+			},
+			wantErr: true,
+		},
+	}
+	for i, tt := range tests {
+		tt := tt
+		t.Run(tt.name, func(t *testing.T) {
+			t.Parallel()
```
Comment (mmmommm) [inline list_lookup_test.go:185]: @sivchari 
Isn't this it?

File: list.go (hunk 1 above)
Comment (mmmommm) [reply list.go:42]: Please running this command, invalid request error return this struct. 
`curl "https://api.twitter.com/2/users/111111111111111111111111/owned_lists" -H "Authorization: Bearer $token"`

File: list_lookup_test.go (hunk 4 above)
Comment (kazdevl) [reply list_lookup_test.go:186]: @mmmommm
you can refer [this discussion](https://github.com/sivchari/gotwtr/pull/44#discussion_r753326996)

Comment (mmmommm) [reply list_lookup_test.go:187]: @kazdevl 
ty :) I fixed

File: list.go (hunk 1 above)
Comment (sivchari) [reply list.go:42]: ty

**Review by sivchari — APPROVED** [review]
LGTM

**Pull request merged**